packages/mc-scripts/config/polyfills.js

```diff
@@ -10,7 +10,7 @@
  *    This is why `unfetch` is still used as a polyfill. Overwriting or
  *    stubbing a ponyfill is not possible as JavaScript modules are read-only.
  */
-import 'unfetch/polyfill';
+import 'unfetch/polyfill/index';
```

Comment:
using the default export means we use the `mjs` file, which has untranspiled code.